skyportal/handlers/api/gcn_tach.py

```diff
@@ -211,13 +212,13 @@ def post_aliases(dateobs, tach_id, user):
             return
 
         if gcn_event.aliases is None:
-            gcn_event.aliases = new_gcn_aliases
+            gcn_event.aliases = list(set(new_gcn_aliases))
         else:
             gcn_aliases = [alias.upper() for alias in gcn_event.aliases]
             for new_gcn_alias in new_gcn_aliases:
                 if new_gcn_alias.upper() not in gcn_aliases:
                     gcn_aliases.append(new_gcn_alias)
```

Comment:
You should probably append the `new_gcn_alias` with an `.upper()` call.

Reply:
Make sure that in all cases what you end up appending into the final list is all uppercase.

Reply:
Hi @guynir42, I made the changes you required, and other modifications as well, as I noticed that the behavior of some if statements to check if we already have something in the DB for the circulars and aliases was incorrect. It should work just fine now. It's a minor change and you already approved the changes, so I'm not tagging you for a new review. Just let me know if you want to check the changes.